tests/unit_tests/test_dqc_us_0004.py

```diff
@@ -124,3 +123,38 @@ def test_values_unequal_very_unequal_values(self):
         self.assertTrue(_values_unequal(220, 100, -1))
         self.assertTrue(_values_unequal(600, 100, -2))
 
+    def test_values_scale_nan(self):
+        """
+        Make sure that the scale value is not valid if it is Not a Number
+        """
+        self.assert_true(_min_dec_valid(float('nan')))
```

Comment:
I think this needs to be assertTrue.  Also, I think NaN would return false.  I am thinking you want to keep all of the original cases you were testing before removing the other two, but need to break it out as follows:

```
self.assertTrue(_min_dec_valid(100))
self.assertTrue(_min_dec_valid(-100))
self.assertFalse(_min_dec_valid(float('nan)))
```

Same would apply for the other tests.  Break it out to make small, unique assertions on each line, making sure you cover all cases.

Ping me if you have any questions.